packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -98,19 +99,24 @@ if [[ '%{release}' != *"develop"* ]];then
   VERSION="%{version}"
   sed "s/nightly/${VERSION%.*}/g" -i %{buildroot}%{repo_dir}/*.repo
   sed "s/gpgcheck=0/gpgcheck=1/g" -i %{buildroot}%{repo_dir}/foreman.repo
-  sed "s/gpgcheck=0/gpgcheck=1/g" -i %{buildroot}%{repo_dir}/foreman-rails.repo
+  sed "s/gpgcheck=0/gpgcheck=1/g" -i %{buildroot}%{repo_dir}/foreman-{rails,client}.repo
```

Comment:
I'd prefer this to be two separate lines (otherwise you could also write {,-rails,-client}, right? ;))

Reply:
Yes, I considered that as well. Do you prefer separate lines or 1 smart line?

Reply:
I find separate lines more readable, as the user does not have to know/remember how bash/zsh does brackets (and that plain posix sh does not ;))

Reply:
Updated